pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -71,19 +85,29 @@ def render(self, task) -> Text:
                 metrics = self._trainer.progress_bar_callback.get_metrics(self._trainer, self._pl_module)
             else:
                 metrics = self._trainer.progress_bar_metrics
+
+            # todo: what even is v_num
+            metrics.pop("v_num", None)
```

Comment:
Can anyone enlighten me on what this is? cc @kaushikb11

Reply:
the version number of the experiment. this info comes from the logger